core/controllers/editor.py

```diff
@@ -659,16 +958,46 @@ class ImageUploadHandler(EditorHandler):
     """Handles image uploads."""
 
     _decorator = None
+    URL_PATH_ARGS_SCHEMAS = {
+        'entity_type': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'entity_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'image': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'filename': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
```

Comment:
Any other rules?

Reply:
For image I have added max_size & for filename I have added regex pattern, please take a look.